### PR TITLE
feat: decrypt document

### PR DESCRIPTION
**Pull request by yehjxraymond**
Added decryption capabilities to QR Processor.

Encrypted resources has `key` in the action payload. When receiving such payload, it will access the resource with the `uri` and then decrypt the payload in `cipherText` with `key`, `type`, `iv` and `tag`.

Test with: 
image

Url in image: https://api.myjson.com/bins/1bpef8
Decryption key in QR: `a93bf5175b276332a9bd4ed43b4ccd2c0968febb18ae1af7f7e5daa30b333e0b`

### CODE REVIEW

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=28

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-28

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=28

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-28

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=28

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-28

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=28

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-28

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=28

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-28

File: src/services/QrProcessor/QrProcessor.test.tsx [hunk 1]
```diff
@@ -71,33 +73,35 @@ describe("decodeAction", () => {
...
 
 describe("processQr", () => {
-  const mockDocument = "MOCK_JSON_RESPONSE" as any;
+  const mockJsonResponse = jest.fn();
 
   beforeAll(() => {
     const globalAny: any = global;
     jest
       .spyOn(globalAny, "fetch")
       .mockImplementation()
```
Comment (sebastianquek) [inline src/services/QrProcessor/QrProcessor.test.tsx]: Is there a need for this additional `mockImplementation`?

**Comment by john-dot-oa** [comment]
## Application
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=28

## Storybook
Expo QR
Published to https://exp.host/@dlt/identitywallet?release-channel=storybook-28

**Pull request merged**